### ISSUE TITLE
col_values() does not work - KeyError: u'\xe1'

### ISSUE THREAD

**Issue by blaskovic** [burnash/gspread#558]
### Environment info

Operating System: Debian 8 and Fedora 24
Python version: Python 2.7.13
gspread version: 3.0.0 (it works in 0.6.2)

### Steps to reproduce
1. Testing on this: https://docs.google.com/spreadsheets/d/1SwKY8fe5OVqsWCpdXtFFBu2EQ14F-SaC-Bywjldf5ls
2. Using just `col_values()`, exact script:

```
#!/usr/bin/python
# -*- coding: utf8 -*-
import gspread
from oauth2client.service_account import ServiceAccountCredentials

scope = ['https://spreadsheets.google.com/feeds']
credentials = ServiceAccountCredentials.from_json_keyfile_name('.secret.json', scope)
gc = gspread.authorize(credentials)
sh = gc.open_by_url('https://docs.google.com/spreadsheets/d/1SwKY8fe5OVqsWCpdXtFFBu2EQ14F-SaC-Bywjldf5ls')
ws = sh.get_worksheet(0)

all_lines = ws.col_values(1)
print all_lines
```

### Stack trace or other output that would be helpful

```
$ python test-gspread.py 
/usr/lib64/python2.7/urllib.py:1299: UnicodeWarning: Unicode equal comparison failed to convert both arguments to Unicode - interpreting them as being unequal
  return ''.join(map(quoter, s))
Traceback (most recent call last):
  File "test-gspread.py", line 14, in <module>
    all_lines = worksheet_vratky.col_values(1)
  File "/usr/lib/python2.7/site-packages/gspread/models.py", line 524, in col_values
    'majorDimension': 'COLUMNS'
  File "/usr/lib/python2.7/site-packages/gspread/models.py", line 109, in values_get
    url = SPREADSHEET_VALUES_URL % (self.id, quote(range, safe=''))
  File "/usr/lib64/python2.7/urllib.py", line 1299, in quote
    return ''.join(map(quoter, s))
KeyError: u'\xe1'
```

**Comment by burnash** [comment]
Looks like this is related to https://github.com/burnash/gspread/issues/544

**Comment by burnash** [comment]
Fixed in #610 and #628
Available in version 3.1.0
